src/pages/Login.jsx

```diff
@@ -1,10 +1,7 @@
-//smart component
 import React from "react";
-import LoginForm from "@features/authentication/components/LoginForm/LoginForm";
-import { useAuth } from "@features/authentication/hooks/useAuth";
+import LoginForm from "@features/authentication/components/LoginForm";
 const Login = () => {
-  const { login } = useAuth();
-  return <LoginForm onLogin={login} />;
+  return <LoginForm />;
```

Comment:
loginForm should only have the logic of the login. The Title should put out of there. So it should be something like this:
```
const LoginPage = () =>{
return (
<S.Container> --- a style from styled components
<Title/> --- this should be a component
<LoginForm/>
<S.Container/>
)
```
}